app/Actions/HandlePurchaseAction.php

```diff
@@ -66,29 +70,38 @@ public function execute(
         }
 
         if ($user->email) {
-            Mail::to($user->email)->queue(new PurchaseConfirmationMail($purchase));
+            foreach ($purchasables as $purchasable) {
+                Mail::to($user->email)->queue(new PurchaseConfirmationMail($purchase, $purchasable));
```

Comment:
Should we send a custom "bundle" purchase confirmation email?

Reply:
Let's keep it simpel and send the mails of the separate products inside the bundle.